lib/LaTeXML/Util/Pack.pm

```diff
@@ -40,7 +40,7 @@ sub unpack_source {
   foreach my $member ($zip_handle->memberNames()) {
     $zip_handle->extractMember($member, catfile($sandbox_directory, $member)); }
   # Set $source to point to the main TeX file in that directory
-  my @TeX_file_members = map { $_->fileName() } $zip_handle->membersMatching('\.tex$');
+  my @TeX_file_members = map { $_->fileName() } $zip_handle->membersMatching('\.[tT][eE][xX]$');
```

Comment:
i would usually use the `/i` flag here, but since it's passed into a method I ended up dual-casing each letter